components/places/src/api/history.rs

```diff
@@ -60,9 +60,9 @@ mod tests {
     use super::*;
 
     #[test]
-    fn test_insert() {
-        let mut c = PlacesDb::open_in_memory(None).expect("should get a connection");
-        let url = Url::parse("http://example.com").expect("it's a valid url");
+    fn test_insert() -> Result<()> {
+        let mut c = PlacesDb::open_in_memory(None)?;
```

Comment:
Last time I tried this, writing the tests in this way removed line numbers from panics. That makes it much harder to track down which assert/expect/unwrap fired. Is that still true?

Reply:
Ouch - it does remove them :( I reverted that.